minio/api.py

```diff
@@ -1558,7 +1559,7 @@ def _do_put_multipart_object(self, bucket_name, object_name, part_metadata,
 
         headers = {
             'Content-Length': part_metadata.size,
-            'Content-Md5': part_metadata.md5hasher.base64digest()
+            'Content-Md5': base64.b64encode(part_metadata.md5_hex.decode("hex"))
```

Comment:
The error happens if you run python3 fput_object.py, i.e. by moving the fput_object.py from examples/ folder to the main repo directory.